examples/configs/neurobooth_os_config.json

```diff
@@ -1,26 +1,28 @@
 {
-    "data_out": "C:\\neurobooth\\neurobooth_data\\\\",
-    "nas": "Z:\\data\\\\",
-    "video_tasks": "C:\\Users\\STM\\Dropbox (Partners HealthCare)\\Neurobooth Videos for tasks\\Videos_to_present",
+    "remote_data_dir": "C:/data/",
+    "video_tasks": "C:/Users/STM/Dropbox (Partners HealthCare)/Neurobooth Videos for tasks/Videos_to_present",
     "cam_inx_lowfeed": 0,
-    "default_log_path": "D:\\neurobooth\\neurobooth_logs",
+    "default_log_path": "C:/neurobooth/neurobooth_logs",
```

Comment:
Should we specify the log path separately for each machine? Was under D:/neurobooth/neurobooth_logs for ACQ (and the equivalent C:/ path for other machines)

Reply:
(No real reason, just that I had decided to place the logs directory as a sibling to the data directory and that's how the data directories were arranged.)

Reply:
I can make it machine specific, but I'm still under the impression that we plan to centralize logging, so then it should go back to being the same for all machines

Reply:
Makes sense